content/en/blog/2023/sig-instrumentation-spotlight.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: blog
+title: "Spotlight on SIG Instrumentation"
+slug: sig-instrumentation-spotlight
+date: 2023-01-31
+canonicalUrl: https://www.kubernetes.dev/blog/2023/01/31/sig-instrumentation-spotlight-2023/
+---
+
+**Author**: Imran Noor Mohamed(Delivery Hero)
```

Comment:
For this site, AIUI we prefer to use the front matter to set an author. See other articles such as https://github.com/kubernetes/contributor-site/blame/master/content/en/blog/2022/sig-storage-spotlight.md#L6 for an example.